### DISCUSSION TITLE
Sell old submarines/dismantle it

### DISCUSSION THREAD

**Discussion by lolwhatik** [FakeFishGames/Barotrauma#11990] [Ideas and suggestions]
During your playthrough you're expected to upgrade your sub, buy new\better\bigger ones, but when u get new one, the previous sub stays in the dock, you've spent your money on it, spent money on upgrades, and now its just there, u cant sell it, u cant dismantle or transfer upgrades etc. would be nice to be able to scap it for money at least, but its up to you guys to figure out what's more to do besides scaping for money.(or we could brainstorm it in discord if u post discussion for it)

**Comment by Ek-Videogames** [comment]
The addition of tier-limited upgrades, and now forced submarine tiers in campaign progression - means there's now even less reason anyone would ever want to switch back to a previous submarine. 

It's been proposed a few times already by various people, but a "sell/trade-in" system would help alleviate the problem of being forced to waste money on intermediary-tiers of submarines and then not being able to afford to upgrade to the next more expensive one.
